using-timescaledb/continuous-aggregates.md

```diff
@@ -160,123 +275,91 @@ something like:
 ```sql
 SELECT * FROM conditions_summary_daily
 WHERE max - min > 1800
-  AND bucket >= '2018-01-01' AND bucket < '2018-04-01'
+  AND bucket >= '2020-01-01' AND bucket < '2020-04-01'
 ORDER BY bucket DESC, device_id DESC LIMIT 20;
 ```
 
+---
+
+### Real-Time Aggregates [](real-time-aggregates)
+
+Real-time aggregates are a capability (first introduced in TimescaleDB 1.7)
+whereby querying the *continuous aggregate view* will then compute
+fully up-to-date aggregate results by combining the materialized
+partial aggregate with recent data from the hypertable that has yet to
+be materialized by the continuous aggregate. By combining raw and
+materialized data in this way, one gets accurate and up-to-date
+results while still enjoying the speedups of pre-computing a large
+portion of the result.
+
+As an example, continuous aggregates _without_ this real-time capability make
+it really fast to get aggregate answers by pre-computing these values (such as
+the min/max/average value over each hour). This way, if you are collecting raw
+data every second, querying hourly data over the past week means reading 24 x 7
+= 168 values from the database, as opposed to processing 60 x 60 x 24 x 7 =
+604,800 values at query time.
+
+But this type of continuous aggregate does not incorporate the very
+latest data, _i.e._, since the last time the asynchronous aggregation job ran
+inside the database. So if you are generating hourly rollups, you might only
+run this materialization job every hour.
+
+With real-time aggregates, a single, simple query will combine your
+pre-computed hourly rollups with the raw data from the last
+hour, to always give you an up-to-date answer.  Now, instead of touching
+604,800 rows of raw data, the query reads 167 pre-computed rows of
+hourly data and 3600 rows of raw secondly data, leading to significant
+performance improvements.
+
+Real-time aggregates are now the default behavior for any continuous
+aggregates. To revert to the previous behavior, in which the query
+touches materialized data only and doesn't combine with the latest raw
+data, add the parameter `timescaledb.materialized_only=true` when
+creating the continuous aggregate view.
+
+You can also use this in conjunction with the `ALTER MATERIALIZED
+VIEW` to turn this feature on or off at any time.
+
+>:TIP: To upgrade continuous aggregates that were created in a version
+earlier than TimescaleDB 1.7 to use real-time aggregates, ALTER the
+view to set `timescaledb.materialized_only=false`.  All subsequent
+queries to the view will immediately use the real-time aggregate
+feature.
+
+---
+
 ### Advanced Usage
```

Comment:
What's the difference between "Advanced Usage" and "Advanced Topics"?

Reply:
There is none. Moved the heading so that there is only one and also removed some stuff and moved some stuff around.